modules/moordyn/src/MoorDyn_Misc.f90

```diff
@@ -1707,49 +1712,58 @@ SUBROUTINE gridAxisCoords(coordtype, entries, coordarray, n, ErrStat, ErrMsg)
          REAL(ReKi) :: tempArray (100)
          REAL(ReKi) :: dx
          INTEGER(IntKi)                   :: nEntries, I
-         
-         ! get array of coordinate entries 
-         CALL stringToArray(entries, nEntries, tempArray)
-         
-         ! set number of coordinates
-         if (     coordtype==0) then   ! 0: not used - make one grid point at zero
-            n = 1;
-         else if (coordtype==1) then   ! 1: list values in ascending order
-            n = nEntries
-         else if (coordtype==2) then   ! 2: uniform specified by -xlim, xlim, num
-            n = int(tempArray(3))
-         else
-            print *, "Error: invalid coordinate type specified to gridAxisCoords"
-         end if
-         
-         ! allocate coordinate array
-         CALL AllocAry(coordarray, n, 'x,y, or z grid points' , ErrStat, ErrMsg)
-         !ALLOCATE ( coordarray(n), STAT=ErrStat) 
-         
-         ! fill in coordinates
-         if (     coordtype==0) then
-            coordarray(1) = 0.0_ReKi
-         
-         else if (coordtype==1) then
-            coordarray(1:n) = tempArray(1:n)
-         
-         else if (coordtype==2) then  
-            coordarray(1) = tempArray(1)
-            coordarray(n) = tempArray(2)
-            dx = (coordarray(n)-coordarray(0))/REAL(n-1)
-            do i=2,n-1
-               coordarray(i) = coordarray(1) + REAL(i)*dx
-            end do
-         
-         else
-            print *, "Error: invalid coordinate type specified to gridAxisCoords" 
-         end if
-         
-         print *, "Set water grid coordinates to :"
-         DO i=1,n
-            print *, " ", coordarray(i)
-         end do
-         
+
+         IF (len(trim(entries)) == len(entries)) THEN
+            print*, "Warning: Only 120 characters read from wave grid coordinates"
```

Comment:
We generally discourage the use of `print*` statements as they do not work when coupled to Simulink.  Instead, use `call WrScr("text to print")`.